File: Iniciante/1008 - Salário/Python/1008.py
def main():
    employee_number = int(input())
    hours_worked = int(input())
    hour_value = float(input())
    salary = hour_value * hours_worked
    print(f"NUMBER = {employee_number}\nSALARY = U$ {salary:.2f}")


if __name__ == "__main__":
    main()

File: Iniciante/1008 - Salário/TypeScript/1008.ts
import * as fs from "fs";

function main(): void {
  const input: string = fs.readFileSync("/dev/stdin", "utf8");
  const lines: string[] = input.split("\n");
  const employeeNumber: number = parseInt(lines.shift()!);
  const hoursWorked: number = parseInt(lines.shift()!);
  const hourValue: number = parseFloat(lines.shift()!);
  const salary: number = hourValue * hoursWorked;
  console.log(`NUMBER = ${employeeNumber}\nSALARY = U$ ${salary.toFixed(2)}`);
}

main();
